app/components/widgets/AlertBox.vue

```diff
@@ -31,7 +31,10 @@
         <span class="left-accordion__title">{{ alertName(alert) }}</span>
       </div>
       <div class="left-accordion__input" v-if="wData.settings[alert]">
-        <validated-form  @input="save()"><toggle-input v-model="wData.settings[alert].enabled" /></validated-form>
+        <validated-form v-if="alert !== 'facemasks'" @input="save()"><toggle-input v-model="wData.settings[alert].enabled" /></validated-form>
```

Comment:
Nitpick: remove brackets.

Reply:
not sure which brackets you're referring to here